crates/uv-fs/src/path.rs

```diff
@@ -34,7 +39,9 @@ impl<T: AsRef<Path>> Simplified for T {
 
     fn user_display(&self) -> std::path::Display {
         let path = dunce::simplified(self.as_ref());
-        path.strip_prefix(&*CWD).unwrap_or(path).display()
+        path.strip_prefix(CWD.simplified())
+            .unwrap_or(path)
+            .display()
     }
 }
```

Comment:
This fixes a bug where sometimes the path was not properly relativized. This is the only user-facing change here and could be split into a separate pull request. It's needed for consistent cross-platform snapshot replacements.